### PR TITLE
catching the case where there are no measurements, so histFlag is emp…

### PR DESCRIPTION
**Pull request by mhines-usgs**
…ty and no length. Setting histFlag to false so it can continue through the measurement function without erroring.

AKA AQCU-883

### CODE REVIEW

File: R/vdiagram-data.R [hunk 1]
```diff
@@ -42,7 +42,9 @@ defaultHistFlags <- function(histFlag){
   if (length(histFlag)==1 && histFlag == " "){
     histFlag <- rep(TRUE, length(x))
   }
-  
+  if (histFlag==""){
```
Comment (lindsayplatt) [inline R/vdiagram-data.R]: I think isEmptyOrBlank will do this

Comment (mhines-usgs) [reply R/vdiagram-data.R]: k, i'll try that

Comment (mhines-usgs) [reply R/vdiagram-data.R]: yup :D :+1:

Comment (lindsayplatt) [reply R/vdiagram-data.R]: just trying to take advantage of all those functions we made :)

Comment (mhines-usgs) [reply R/vdiagram-data.R]: agreed, thanks for being vigilant :sheriffs-badge:

**Pull request merged**